CHANGELOG.md

```diff
@@ -12,6 +12,26 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+### Deprecations
+
+### New features
+
+### Fixed bugs
+
+## [0.28.0] - 2025-04-02
+
+### Release highlights
+
+* All outstanding "pinned" issues in the GitHub repository have been resolved, including a long-standing bug relating to empty files in the built-in diff editor.
```

Comment:
nit: wrap at ~80 columns (we should set up a formatter. i don't remember what the latest on that is)

Reply:
Setting up a formatter is related to https://github.com/jj-vcs/jj/pull/6081: would `mise` make it easier for end-users to install `dprint` or `prettier`? (IIRC those were the most promising formatters for Markdown; even if we go with `prettier`, it might be easier to run it via `dprint`)

We could then run the formatter with either `mise` (or equivalent) or with `jj fix`.